.editorconfig

```diff
@@ -0,0 +1,28 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+
+[vcbuild.bat]
+end_of_line = crlf
+
+[*.{md,markdown}]
+trim_trailing_whitespace = false
```

Comment:
Yes, trailing space to force a line break in markdown. Not a fan of it either, but we should leave that option available to collaborators.